src/debug.h

```diff
@@ -53,6 +53,12 @@
 #define __FUNCTION_NAME__ __func__
 #endif
 
+#define fatal(cond) \
```

Comment:
fatal() is probably a poor choice of name here since it is fatal if the condition is false, not if it is true.